bitnami/fluentd/templates/aggregator-statefulset.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "fluentd.fullname" . }}
+  labels:
+    {{- include "fluentd.labels" . | nindent 4 }}
+    app.kubernetes.io/component: aggregator
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "fluentd.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: aggregator
+  serviceName: {{ include "fluentd.fullname" . }}-headless
+  replicas: {{ .Values.aggregator.replicaCount }}
+  updateStrategy:
+    {{- toYaml .Values.aggregator.updateStrategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "fluentd.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: aggregator
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/aggregator-configmap.yaml") . | sha256sum }}
+        {{- if .Values.aggregator.podAnnotations }}
+        {{- toYaml .Values.aggregator.podAnnotations | nindent 8 }}
```

Comment:
`nindent` -> `indent`

Reply:
Why? I'd prefer to keep using `nindent` so that the line can be indented in the template for ease of read while being still correctly indented in the final result.

Reply:
Agree to leave it as nindent